### PR TITLE
PIM-4744: add extra information in the notifications

### PR DESCRIPTION
**Pull request by nidup**
Add extra information in notification as action type and created at to prepare the style revamp

| Q                  | A
| ------------------ | ---
| Migration script?  | n
| Behats             | y
| Specs & Static     | y
| Changelog updated  | n

### CODE REVIEW

File: spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php [hunk 1]
```diff
@@ -72,7 +72,8 @@ function it_notifies_a_user_of_the_completion_of_job_execution($event, $user, $m
                 [
                     'route' => 'pim_importexport_export_execution_show',
                     'routeParams' => ['id' => 5],
-                    'messageParams' => ['%label%' => 'Product export']
+                    'messageParams' => ['%label%' => 'Product export'],
+                    'context' => ['actionType' => 'export']
```
Comment (phaseinducer) [inline spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php]: align please

File: spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php [hunk 2]
```diff
@@ -90,7 +91,8 @@ function it_notifies_a_user_of_the_completion_of_a_mass_edit_job_execution($even
                 [
                     'route' => 'pim_enrich_job_tracker_show',
                     'routeParams' => ['id' => 5],
-                    'messageParams' => ['%label%' => 'Product mass edit']
+                    'messageParams' => ['%label%' => 'Product mass edit'],
+                    'context' => ['actionType' => 'mass_edit']
```
Comment (phaseinducer) [inline spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php]: same

File: spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php [hunk 3]
```diff
@@ -118,7 +120,8 @@ function it_notifies_a_user_of_the_completion_of_job_execution_which_has_encount
                 [
                     'route' => 'pim_importexport_export_execution_show',
                     'routeParams' => ['id' => 5],
-                    'messageParams' => ['%label%' => 'Product export']
+                    'messageParams' => ['%label%' => 'Product export'],
+                    'context' => ['actionType' => 'export']
```
Comment (phaseinducer) [inline spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php]: same

File: spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php [hunk 4]
```diff
@@ -142,7 +145,8 @@ function it_notifies_a_user_of_the_completion_of_job_execution_which_has_encount
...
```
Comment (phaseinducer) [inline spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php]: well... =)

File: src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php [hunk 5]
```diff
@@ -129,6 +130,9 @@ protected function generateExportImportNotify(JobExecution $jobExecution, $user,
                 ],
                 'messageParams' => [
                     '%label%' => $jobExecution->getJobInstance()->getLabel()
```
Comment (willy-ahva) [inline src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php]: $jobInstance->getLabel() now :)

File: src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php [hunk 6]
```diff
@@ -153,6 +157,9 @@ protected function generateMassEditNotify(JobExecution $jobExecution, $user, $ty
...
```
Comment (willy-ahva) [inline src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php]: $jobInstance->getLabel() now :)

File: spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php (hunk 1 above)
Comment (jmleroux) [inline spec/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifierSpec.php]: alignment please

File: features/mass-action/mass_edit_variant_group_products.feature [hunk 7]
```diff
@@ -68,8 +68,8 @@ Feature: Apply restrictions when mass editing products with variant groups
     And I wait for the "add-to-variant-group" mass-edit job to finish
     Then I should have 1 new notification
     And I should see notification:
-      | type    | message                          |
-      | warning | Mass edit finished with warnings |
+      | type    | message                                                 |
+      | warning | Mass edit "Mass update products" finished with warnings |
```
Comment (phaseinducer) [inline features/mass-action/mass_edit_variant_group_products.feature]: Why double quotes only here and not in other messages ?

File: src/Pim/Bundle/NotificationBundle/Resources/views/Notification/list.json.twig [hunk 8]
```diff
@@ -5,7 +5,10 @@
         type: item.notification.type,
         message: item.notification.message|trans(item.notification.messageParams),
         viewed: item.viewed,
-        url: item.notification.route ? path(item.notification.route, item.notification.routeParams) : null
+        url: item.notification.route ? path(item.notification.route, item.notification.routeParams) : null,
+        createdAt: item.notification.created|date("Y-m-d H:i:s"),
+        actionType: item.notification.context.actionType is defined ? item.notification.context.actionType : null,
```
Comment (skeleton) [inline src/Pim/Bundle/NotificationBundle/Resources/views/Notification/list.json.twig]: shortcut: `actionType: item.notification.context.actionType|default(null)`

File: src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php (hunk 5 above)
Comment (phaseinducer) [inline src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php]: :+1:

**Comment by phaseinducer** [comment]
GTM after fixes

File: features/mass-action/mass_edit_variant_group_products.feature (hunk 7 above)
Comment (nidup) [inline features/mass-action/mass_edit_variant_group_products.feature]: typo, i remove them

File: src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php (hunk 5 above)
Comment (nidup) [inline src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php]: in another method

File: src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php (hunk 6 above)
Comment (nidup) [inline src/Pim/Bundle/NotificationBundle/EventSubscriber/JobExecutionNotifier.php]: same

**Pull request merged**